docs/dictionary/function/screenDepth.lcdoc

```diff
@@ -34,4 +34,6 @@ is equal to
 
 The value returned by the <screenDepth> <function> is updated only when you start up the <application>. If you change the screen settings after starting up the <application>, you must quit and restart to update the <screenDepth>.
 
+In  noui mode, the screendepth is 0.
```

Comment:
Didn't bother you the first time :-P (https://github.com/livecode/livecode/pull/3342)

I agree though, will change it.